### PR TITLE
GetInfo use case implemented

### PR DESCRIPTION
**Pull request by quiz3**
I implemented the GetInfo use_case but I also modified a bunch of files cuz the switch to Maven format messed with all the imports so I needed to change a whole bunch of stuff.

Very important to thoroughly review this one though.

This is linked to Issue #29.

### CODE REVIEW

**Review by awesominat — APPROVED** [review]
I really like this, the public methods are documented, no CA engine violations, and a good choice of what to show the user for the GetInfo case.

I think this might go in the Buy Menu after a user searches for a specific stock so they can make sure this is the stock they wanted to buy

**Comment by awesominat** [comment]
I will merge this as soon as we have the new JSON data loader merged

File: src/main/java/use_cases/GetInfo/GetInfoInteractor.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    @Override
+    public void execute(GetInfoInputData getInfoInputData) {
+        /**
+         * The getInfoInputData parameter should follow the specifications laid out in that class.
+         * <p>
+         * This method implements the bulk of the GetInfo use case.
+         * Info is fetched using the Finnhub driver.
+         *
+         * @param  getInfoInputData  an InputData object following the relevant CA Engine rules
+         */
```
Comment (awesominat) [inline src/main/java/use_cases/GetInfo/GetInfoInteractor.java:24]: I feel that the documentation needs to mention what the use case does, or rather, what the execute method does. currently, it just says it does the bulk of the use case, what does that mean?

Also, I don't think you need to mention it follows CA engine rules, the client doesn't really care about that. Instead, mention that the InputData contains the ticker

**Review by awesominat — CHANGES_REQUESTED** [review]
I think the documentation can be improved. this should help clear up any confusion about the doc in the future. very good start though

**Comment by quiz3** [comment]
I have been informed that it is not bad that this is failing. It is likely due to the fact that this branch was created prior to some of the newer Maven architecture of our project being set up. 

The merge process for this will be non-trivial but not very difficult, since all I modified most recently was the interaction.

**Review by awesominat — APPROVED** [review]
lgtm, thanks for the documentation changes

**Pull request merged**